pages/about.html

```diff
@@ -1,12 +1,87 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta http-equiv="X-UA-Compatible" content="IE=edge">
-  <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Document</title>
-</head>
-<body>
-  
-</body>
-</html>
\ No newline at end of file
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>About me</title>
+    <link href="../styles/style.css" rel="stylesheet" />
+    <link href="../styles/about.css" rel="stylesheet" />
+  </head>
+  <body>
+    <nav>
+      <div class="logo">
+        <img src="/assets/logo.png" alt="Website-logo" />
+      </div>
+      <ul>
+        <li>
+          <a href="index.html"><em>00.</em>Home</a>
+        </li>
+        <li>
+          <a href="about.html"><em>01.About</em></a>
+        </li>
+        <li>
+          <a href="portfolio.html"><em>02.</em>Projects</a>
+        </li>
+        <li>
+          <a href="contact.html"><em>03.</em>Contact</a>
+        </li>
+      </ul>
+    </nav>
+    <div class="contact-container">
+      <ul class="contact-list">
+        <li>
+          <a href="https://github.com/diakymenko" target="_blank">GitHub</a>
+        </li>
+        <li>
+          <a
+            href="https://www.linkedin.com/in/daria-iakymenko-26177831"
+            target="_blank"
+            >LinkedIn</a
+          >
+        </li>
+        <li>
+          <a href="mailto:daria.iakymenko@gmail.com">Email</a>
+        </li>
+      </ul>
+    </div>
+    <div class="main">
```

Comment:
Consider using a different class name here. Using a class named `main` is confusing, because there's also a `<main>` tag in HTML.